3d/commands.py

```diff
@@ -0,0 +1,24 @@
+#   Copyright 2016 Peter Kropf
```

Comment:
I'm ok with changing the copyright.


On Fri, Aug 19, 2016 at 11:06 AM, Scott Bezek <notifications@github.com>
wrote:

> In 3d/commands.py
> <https://github.com/scottbez1/splitflap/pull/13#discussion_r75526457>:
>
> > @@ -0,0 +1,24 @@
> > +#   Copyright 2016 Peter Kropf
>
> I think for consistency (and scalability) it might make sense to switch to
> a standardized header for all files in the repo, something like:
> "Copyright 2015-2016 Scott Bezek and the splitflap contributors"?
>
> I can go ahead and send a separate diff that will update the existing
> files, README, LICENSE, etc, to match, and that way it will be clear that
> the code in this repo is not from a single person. As far as per-file
> authorship goes, I think git history is probably a reasonable way to
> preserve attribution since it's automatic and avoids manual churn of header
> comments as new authors contribute/modify the code.
>
> I'm largely going off the discussion here http://ben.balter.com/2015/06/
> 03/copyright-notices-for-websites-and-open-source-projects/ but open to
> alternative suggestions.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/scottbez1/splitflap/pull/13/files/27a01659e6c0123c91213a408cf76f70c955576b#r75526457>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAGEE76sZeghb9WKso7u8QZB8BM71zk7ks5qhfCggaJpZM4JoFys>
> .
>